weighted_levenshtein/clev.pyx

```diff
@@ -32,7 +22,7 @@ ctypedef struct Array2D:
 
 cdef inline void Array2D_init(
     Array2D* array2d,
-    Py_ssize_t num_rows,
+    Py_ssize_t num_rows, 
```

Comment:
nit: Please do not add in these unnecessary spaces (also applies to many of the lines below in this file)